### PR TITLE
Release candidate missing themes

### PR DESCRIPTION
**Pull request by neil-stephen-mcgonigle**
adding in missing themes

### CODE REVIEW

File: src/lib/resource/sections/mainapplicant-enter-your-name.js [hunk 1]
```diff
@@ -4,47 +4,90 @@ module.exports = {
...
-                    title: 'Last name',
-                    type: 'string',
-                    maxLength: 70,
+            allOf: [
+                {
+                    title: 'Enter your name',
+                    meta: {
+                        compositeId: 'mainapplicant-name',
+                        classifications: {
+                            theme: 'main-applicant-details'
```
Comment (armoj) [inline src/lib/resource/sections/mainapplicant-enter-your-name.js:14]: should the theme be consistent with the ids/compositeIds and have no hyphen between `main` and `applicant`? More a question than a required change

**Review by armoj — CHANGES_REQUESTED** [review]

File: src/lib/resource/sections/mainapplicant-enter-your-name.js (hunk 1 above)
Comment (neil-stephen-mcgonigle) [reply src/lib/resource/sections/mainapplicant-enter-your-name.js:14]: they probably should

**Comment by neil-stephen-mcgonigle** [comment]
Proposed bump to 5.2.3

File: src/lib/resource/sections/mainapplicant-enter-your-name.js (hunk 1 above)
Comment (neil-stephen-mcgonigle) [reply src/lib/resource/sections/mainapplicant-enter-your-name.js:14]: And the themes are distinct from p and q ids

**Review by armoj — APPROVED** [review]

**Pull request merged**